draft-ietf-core-cf-reg-update.md

```diff
@@ -57,6 +57,7 @@ entity:
 
 This document updates RFC7252 regarding the registration procedures for the "CoAP Content-Formats" IANA registry, within the "Constrained RESTful Environments (CoRE) Parameters" registry group.
 This document also introduces a new column, "Media Type", to the registry.
+Furthermore, this document retires 64999 from the allocatable Content-Format identifiers and reserves its use for documentation.
```

Comment:
Can we say this simpler and shorter? If 64999 is reserved for documentation, we don't have to say that it's retired. It's anyway not usable for allocation if it already is reserved for something. E.g.:

```suggestion
Furthermore, this document allocates Content-Format identifier 64999 for use in documentation.
```

Reply:
What was the reason we are not using 65000?

Reply:
@cabo We use the number in examples of a person requesting a registration within the "FCFS range" of the registry. To make the example convincing, it has to lie within this range. (I.e. in the "old" situation prior to the changes by this document.)
Also the experimental value 65000 might be in actual use in experiments, while 64999 is not supposed to be in use anywhere.

Reply:
Using a code point just for demonstrating how to use a code point causes cognitive dissonance.
So why don't we use 20000?  That also is allocated and FCFS.